src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+    Hello, world!
+  </body>
+</html>
```

Comment:
![2](https://user-images.githubusercontent.com/66570142/111592415-6c4f8f00-87d1-11eb-861b-3124ebc59715.png)

Let's add an empty line at the end of your code